src/abstractsparsearrayinterface.jl

```diff
@@ -357,11 +357,13 @@ function sparse_mul!(
   β::Number=false;
   (mul!!)=(default_mul!!),
 )
+  storage(a_dest) .*= β
+  β′ = true
```

Comment:
```suggestion
  β′ = one(Bool)
```
I know I used `true` and `false` other places, but I think we should switch over to this style, since it shows the intent of the code better. I remember the first time I saw `true` and `false` used as unopinionated one and zero values I was very confused, but I couldn't think of a better alternative. At some point Miles suggested the style `one(Bool)` and `zero(Bool)` as an alternative, which I think is more readable.

Reply:
Thanks for the reminder, I was vaguely aware of some funny business like that, that is some pretty crazy behavior... Agreed it makes more sense to have specialized one and zero types for this kind of thing.

Reply:
(You resolved the conversation but I didn't see it changed to `β′ = one(Bool)`.)

Reply:
Apologies, I did not mean to do that. Should now be resolved